scripts/mkcloud

```diff
@@ -134,6 +136,7 @@ fi
 pidfile=mkcloud.pid
 
 trap 'error_exit $? "error caught by trap"' TERM
+exec 3<&0
```

Comment:
we are already leaking the input fd to several lvm commands (see the mkcloud log files)

So I would recommend to not mess with it even more (at least not for the default mkcloud runs). I would be fine with adding a crowbar-development mode (something like: `export CROWBARDEV=1`).